fetch_submissions.py

```diff
@@ -141,13 +145,18 @@ def save_submission_as_file(submission):
 
 
 if __name__ == '__main__':
-    if (len(sys.argv) != 2):
-        print('Usage: python3 fetch_submissions.py <username>')
+    is_logged_in = False
+    for i in range(5):
```

Comment:
I think we can replace `i` with either `_` (since it is not used anywhere) or with `retry_count`.